### PR TITLE
Update user map for Dec 2017 #8292

### LINKED ISSUE #8292: Update user map for Dec 2017
Similar to #7424 

Date: End of Dec 2017

Entries to consider adding

Australia | University of New South Wales
-- | --
Belgium | Erasmushogeschool Brussel
Belgium | Hasselt University
Belgium | Howest
Belgium | UNamur
Belgium | Universiteit Antwerpen
Belgium | Universiteit Antwerpen
Brazil | Federal University of Bahia
Canada | Canadian Urban Transit Research and Innovation Consortium
Canada | Carleton University
Canada | McGill University
Canada | New York Institute of Technology
Canada | New York Institute of Technology
Canada | Queen's University
Canada | Sir William Gage Middle School
Canada | University of Lethbridge
Canada | University of Toronto Mississauga
Canada | University of Waterloo
Canada | University of Windsor
Colombia | Cooperative University of Colombia
France | EM Lyon
France | Grenoble Ecole de Management
France | IESEG
France | IESEG
France | sup'biotech
France | EM Lyon
Greece | University of Applied Sciences of Thessaly
India | National Institute of Technology, Uttarakhand
Indonesia | Institut Teknologi Bandung
Indonesia | Universitas Padjadjaran
Ireland | University college dublin
Ireland | University College of Dublin
Italy | Bocconi University
Korea | Solbridge International School of Business
Malaysia | International IslamIc University Malaysia
Malaysia | Universiti Teknologi Petronas
Netherlands | Eindhoven University of Technology
Netherlands | Fontys
Netherlands | Juridische Hogeschool Avans&Fontys
Netherlands | NHTV
Netherlands | Vrije Universiteit Amsterdam
New Zealand | University of Auckland
Philippines | University of St. La Salle
Portugal | School of Education Porto (ESE)
Singapore | Nanyang Technological University
Singapore | Singapore Institute of Technology
Spain | University of Deusto
UK | Bradford
UK | University of Arkansas at Little Rock
UK | University of Bath
UK | University of Portsmouth
USA | Arizona State University
USA | Austin Community College
USA | Baylor University
USA | Bridgewater State University
USA | Cabrini University
USA | California State University San Marcos
USA | Central Oregon Community College
USA | Davenport University
USA | Drexel University
USA | Florida International University
USA | Florida State University
USA | Georgetown University
USA | Golf Platoon
USA | Granada Middle School
USA | Harvard Extension School
USA | Las Positas College
USA | Lebanon Valley College
USA | Montana State University
USA | Navasota High School
USA | North Carolina A&T State University
USA | Northeastern University
USA | Northeastern University
USA | Providence College
USA | Riverview Opportunity Center
USA | Siena College
USA | The New School
USA | University at Albany
USA | University at Buffalo
USA | University of Alabama at Birmingham
USA | University of Bridgeport
USA | University of California Santa Barbara
USA | University of Colorado, Boulder
USA | University of Delaware
USA | University of Florida
USA | University of Illinois
USA | University of Iowa
USA | University of Minnesota - Carlson
USA | University of North Carolina
USA | University of North Carolina Greensboro
USA | University of Northern Arizona
USA | University of Virginia
USA | University of Washington
USA | University of Washington
USA | Virginia Tech
USA | Walnut Hills High School
USA | Washington State University
USA | Weber State University
USA | West Chester University
USA | William Woods Univeristy
USA | Xavier University

### PR DESCRIPTION
**Pull request by dishant-sheth**
I wanted to resolve #8292 and hence I have updated the usermap JSON file.

Fixes #8292

### CODE REVIEW

File: src/main/webapp/data/userMapData.json [hunk 1]
```diff
@@ -1035,6 +1052,7 @@
       "The University of Manchester",
       "UCLU Malaysian Society",
       "University College London",
+      "University of Arkansas at Little Rock",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: This seems to be in USA.

File: src/main/webapp/data/userMapData.json [hunk 2]
```diff
@@ -1321,8 +1342,10 @@
       "Glendale Community College",
       "Glenpool Public School",
       "Goldenwest College",
+      "Golf Platoon",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: This name doesn't tell us which military institute it's from - we might want to exclude it.

File: src/main/webapp/data/userMapData.json [hunk 3]
```diff
@@ -1797,6 +1830,7 @@
       "University of North Carolina at Greensboro",
       "University of North Florida",
       "University of North Texas",
+      "University of Northern Arizona",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Same as `Northern Arizona University`.

**Comment by dishant-sheth** [comment]
The asked changes have been made. Thanks.

File: src/main/webapp/data/userMapData.json [hunk 4]
```diff
@@ -376,7 +376,7 @@
       "Lille Catholic University",
       "Sciences Po",
       "SKEMA Business School",
-      "SUP'biotech",
+      "Sup'biotech",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Capital B

**Comment by dishant-sheth** [comment]
Fixed.

**Pull request merged**